code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubClient.java

```diff
@@ -51,7 +53,7 @@
         HttpResponse response = Request.Post(GITHUB_URL.get()).addHeader("Accept", "application/json").bodyForm(requestData).execute().returnResponse();
         int statusCode = response.getStatusLine().getStatusCode();
         if (statusCode != 200) {
-            throw new ClientVisibleException(statusCode);
+            throw new ClientVisibleException(ResponseCodes.SERVICE_UNAVAILABLE);
```

Comment:
For all these 503s I think we discussed before the code should be something like "GitHubUnavailable" to indicate github is the issue, not Rancher